torch/distributed/benchmarks/README.md

```diff
@@ -0,0 +1,68 @@
+# Benchmark combining Distributed Data Parallel and Distributed RPC
```

Comment:
nit: since we might add other files to the benchmark directory, maybe we should create a ddp_rpc directory under this folder and put the README and the benchmark file under that folder.